app/packages/registration-schema/validations.ts

```diff
@@ -58,3 +67,24 @@ export function buildValidation(groups: SchemaBlockGroup[], node?: NodeModel) {
     });
     return ret;
 }
+
+export function buildMetadataValidations() {
+    const validationObj: ValidationObject<DraftRegistration> = {};
+    const notBlank: ValidatorFunction[] = [validatePresence({
+        presence: true,
+        ignoreBlank: true,
+        allowBlank: false,
+        allowNone: false,
+        message: translations.validationErrors.blank,
+    })];
+    set(validationObj, DraftMetadataProperties.Title, notBlank);
+    set(validationObj, DraftMetadataProperties.Description, notBlank);
+    set(validationObj, DraftMetadataProperties.License, notBlank);
+
+    const pickOne: ValidatorFunction[] = [validateLength({
+        min: 1,
+        message: translations.validationErrors.mustSelectMinOne,
+    })];
+    set(validationObj, DraftMetadataProperties.Subjects, pickOne);
```

Comment:
Subjects doesn't need validation AFAIK. In the current setup of subject picker, it doesn't really use the changeset for setting the subjects, so the validation wouldn't work.